content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md

```diff
@@ -0,0 +1,29 @@
+Title: Announcing the winner of "Kenneth Gonsalves Award 2016"-Dr.Ajith Kumar B.P!
+Date: 2016-09-16 19:00
+Category: 2016
+Slug: Announcing-the-winner-of-"Kenneth_Gonsalves_Award_2016"-Dr.Ajith_Kumar_B.P!
+Summary: Announcing the winner of "Kenneth Gonsalves Award 2016"-Dr.Ajith Kumar B.P
+
+
+**PSSI is happy as well as feeling honoured to announce that "Dr.Ajith Kumar B.P", a scientist at Inter University Accelerator Centre, as the winner of "Kenneth Gonsalves Award 2016".**
+
+### Dr.Ajith Kumar B.P
+
+
+![Dr.Ajith Kumar B.P](https://in.pycon.org/2015/images/ajith.jpg)
+
+ Ajith Kumar B.P. has been working as a scientist, at Inter University Accelerator Centre, New Delhi, since 1987. He has done his M.Sc and PhD in Nuclear Physics from University of Calicut and M.Sc in Accelerator Physics, from University of Manitoba.  His work mainly involved in the design and development of particle accelerators and associated experimental facilities.
+
+For the past two decades, he has been involved in promoting Free Software tools in Education, as well as putting some efforts to make free and open source tools popular among the academic community , which include Python in their syllabus, and has also written a book on Python"Python for Education" for this purpose.
```

Comment:
This one sentence is too long but grammatically correct. If possible split it up.